index.js

```diff
@@ -149,7 +149,7 @@ const preloadResources = opt => {
     if (/^data:|blob:/i.test(responseUrl)) return;
     const ct = response.headers()["content-type"] || "";
     const route = responseUrl.replace(basePath, "");
-    if (/^http:\/\/localhost/i.test(responseUrl)) {
+    if (/^http:\/\/localhost/i.test(responseUrl) || ct.includes("json")) {
```

Comment:
this would have other effects, except what you want. You will need to rewrite this branch of logic

Reply:
It is working like I want. All the logic under this block is just checking if the resource is an image/css/js/json file under localhost. So this one just matches with the json bit.

Reply:
unless we add another option to the config? cacheThirdPartyAjaxRequests?

Reply:
actually, I would assume at first tha thet `cacheAjaxRequests` option would already included 3rd-party requests (all ajax requests, not just local).

Reply:
I could also just move the json part outside of this `if localhost` block, into its own but just repeat `uniqueResources` stuff:

`
const responseUrl = response.url();
if (uniqueResources.has(responseUrl)) return;
...
uniqueResources.add(responseUrl);
`

Reply:
Yes code will work right, I was confused a bit. The issue here is that code we lose clarity a bit.

before

```
if () {
// localhost
} else {
// 3rd-party
}
```

after

```
if () {
// localhost or 3rd-party
} else {
// 3rd-party
}
```

You see what I mean? I guess we can move out the condition about JSON

Reply:
Yes, you're right, I will make the change. 👌